test/run

```diff
@@ -93,7 +109,8 @@ def add_e2e_tests(litani, test_dir, counter, output_dir):
             ci_stage="test",
             description=f"{test_file.stem}: init",
             outputs=run_dir / ".litani_cache_dir",
-            cwd=run_dir)
+            cwd=run_dir,
+            timeout=2 if fast else 0)
```

Comment:
I'm not a big fan of repeating the `timeout=2 if fast else 0` logic four times, can we set `test_timeout=2 if fast else None` and use that variable instead?

Reply:
Oh I'm noticing that the documentation was updated to reflect the current timeout of those tests. I'm curious as to why we're being so strict about the timeout on them as they stand now. If we want to add more tests in the future do we want to make it so that they also have to maintain that same pattern of timeouts?

Reply:
I updated all timeouts to be 10s like you suggested.
It is totally up to us to be strict and pass timeout as well with jobs to be sure that for any test which takes longer than the timeout, SLOW is set to True